assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp/index.stories.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * AdBlockingRecoveryApp Component Stories.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import WithRegistrySetup from '../../../../../../../tests/js/WithRegistrySetup';
+import { provideSiteInfo } from '../../../../../../../tests/js/utils';
+import {
+	ENUM_AD_BLOCKING_RECOVERY_SETUP_STATUS,
+	MODULES_ADSENSE,
+	AD_BLOCKING_RECOVERY_SETUP_CREATE_MESSAGE_CTA_CLICKED,
+} from '../../../datastore/constants';
+import { CORE_UI } from '../../../../../googlesitekit/datastore/ui/constants';
+import AdBlockingRecoveryApp from '.';
+
+const Template = () => <AdBlockingRecoveryApp />;
+
+export const StepOne = Template.bind( {} );
+StepOne.storyName = 'Step 1';
+StepOne.args = {
+	setupRegistry: ( registry ) => {
+		registry.dispatch( MODULES_ADSENSE ).setSettings( {
+			adBlockingRecoverySetupStatus: '',
+		} );
+	},
+};
+StepOne.scenario = {
+	label: 'Global/AdBlockingRecoveryApp/StepOne',
```

Comment:
Nitpick: `StepOne` can be misinterpreted as a component here. The last segment should be more of a human-readable label of the scenario. This is consistent with what we have now so this is more of a note for a holistic change since there are many stories that use this now.

These should not however be categorized under "Global" but instead under `Modules/AdSense`.

Reply:
Updated the labels to be more human-readable, let me know if that looks good, thank you!